docker-compose.yml

```diff
@@ -401,6 +402,8 @@ services:
 
       dataProviderUrl: "http://poa-ganache:8545"
       dataRegistryAddress: "${POA_REGISTRY_ADDRESS}"
+
+      autoUpgradeEnabled: "false"
```

Comment:
@dmanjunath is this now the correct place to set stuff?

Reply:
no need to set this for dev since config.js defaults it to false?
also historically for dev we have used https://github.com/AudiusProject/audius-protocol/blob/dm-restructure-fetchCIDInNetwork/creator-node/compose/env/base.env but i think we are maybe moving away from that? i dont remember

Reply:
this is the right place to put stuff for the new local compose tooling. @SidSethi you're right that it's redundant to have here and config.js and base.env but i think it's fine. we should tackle unifying all the places we set configs in a separate task but i think for this task seems okay